pkgs/tools/security/pinentry/default.nix

```diff
@@ -35,29 +36,29 @@ let
     gnome3 = { bin = "gnome3"; flag = "gnome3"; buildInputs = [ gcr ]; nativeBuildInputs = [ wrapGAppsHook ]; };
     qt = { bin = "qt"; flag = "qt"; buildInputs = [ qtbase ]; nativeBuildInputs = [ wrapQtAppsHook ]; };
     emacs = { bin = "emacs"; flag = "emacs"; buildInputs = []; };
+    efl = { bin = "efl"; flag = "efl"; buildInputs = [ pkgs.enlightenment.efl ]; };
   };
 
 in
 
 pinentryMkDerivation rec {
   pname = "pinentry";
-  version = "1.1.0";
+  version = "ae584040d14a4d7d6eed8be77844126130ef0118";
 
-  src = fetchurl {
-    url = "mirror://gnupg/pinentry/${pname}-${version}.tar.bz2";
-    sha256 = "0w35ypl960pczg5kp6km3dyr000m1hf0vpwwlh72jjkjza36c1v8";
+  src = builtins.fetchGit {
```

Comment:
At the moment, fixed-output derivations are still required so you need to use `fetchgit` with `sha256`.